apollo-integration/build.gradle

```diff
@@ -26,8 +26,9 @@ android {
 }
 
 dependencies {
-  provided project(':apollo-api')
+  compile project(':apollo-api')
```

Comment:
can be `provided` because otherwise it would cause dex errors when the plugin is added.